<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="LabelNome">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtNome">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="Dia1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1 dia</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Dia7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>7 dias</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Dia14">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>14 dias</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Dia21">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>21 dias</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Dia30">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>30 dias</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnReserva">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reservar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
